README.md

```diff
@@ -267,6 +267,9 @@ export default DS.JSONAPIAdapter.extend(DataAdapterMixin, {
 });
 ```
 
+**If you set the authorizer to authorizer:application** you need to create an `authorizers/application.js` inside your application (app/authorizers/applications.js). See the following section on implementing a custom authorizer.
+
```

Comment:
Any comments on my replies @marcoow ? I want to help, although I maybe failing at that :)